rootston/output.c

```diff
@@ -333,6 +336,9 @@ static bool has_standalone_surface(struct roots_view *view) {
 	case ROOTS_XWAYLAND_VIEW:
 		return wl_list_empty(&view->xwayland_surface->children);
 #endif
+	case ROOTS_INPUT_PANEL_VIEW:
+		printf("Stub: has_standalone_surface on OSK_VIEV returns true\n");
+		return true;
```

Comment:
The printed text should be updated to correspond to the constant. There is also a typo in the current text (s/OSK_VIEV/OSK_VIEW/).

Reply:
What's a standalone surface here?

Reply:
It's used for fullscreen to know if the view has only one surface.